lib/features/auth/provider/auth_provider.dart

```diff
@@ -92,10 +93,7 @@ class AuthProvider extends StateNotifier<AuthState> {
       return;
     }
     stopListeningSocket();
-    CustomSnackbar.showSnackBar(
-      ref.read(routerProvider).context,
-      'Se ha cerrado sesion exitosamente.',
-    );
+    ref.read(routerProvider).router.push(OnBoarding.route);
```

Comment:
Navigator.of(context).popUntil((route) => route.isFirst);
Navigator.of(context).pushReplacementNamed(OnBoarding.route);